data/obs-packaging/clear-containers-image/README.md

```diff
@@ -0,0 +1,10 @@
+#clear-containers image
+
+This source rpm is the mini-os image used to boot clear-containers. 
+To update and generate an rpm spec and dsc to create rpm and deb packages, 
```

Comment:
Just as above. This sentence makes no sense. I suggest one of two paths:
1. Explicitly list the actions and results of the script.
2. Change the sentence to a simple high-level description with one action with one result.